frontend/src/pages/Inspection/ViewMetadataBody.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { MetaData } from './MetaData';
+// import { Comments } from './Comments';
+
+const Wrapper = styled.div`
+  flex: 1;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+`;
+
+export const ViewMetadataBody = (props) => {
+  const { id } = props;
+  const [data, setData] = useState({
+    uuid: id,
+    url: '',
+    formatName: '',
+    releaseState: 0,
+    metadataTypeName: '',
+    municipalityName: '',
+  });
+  // const [comments, setComments] = useState([]);
+  const [tags, setTags] = useState([]);
+  const [description, setDescription] = useState('');
+
+  useEffect(() => {
+    const internal = async () => {
+      const res = await fetch(`/api/metadata/${id}`);
+      const municipality = await res.json();
```

Comment:
This (line 31 and 32) should be inside a try catch to handle 500 error, and check res.ok and reses.status to make sure the response was okey. Dispatch an alert on errors to give the user feedback